PubbyStation-Reinstallment Initiative

cursedkiller opened the pull request:
## About The Pull Request
Readds the Pubbystation and Relevant Assets also Shifts the Existing Pubby Whiteship into Birdboat while introducing a new one

changed shuttle DM files and Added pubbystation (50=>)  to map rotation aswell as 2 new ships
-New Exploration Shuttle
-MK7 Zealot Cruiser (Whiteship)
-New PubbyStation Map

## Why It's Good For The Game
More maps are always Nice and Pubby was a Fan Favourite for quite some time

## Testing Photographs and Procedure
<!-- Include any screenshots/videos/debugging steps of the modified code functioning successfully, ideally including edge cases. -->
**Everything else is mostly the same as the OG station and hasnt been edited aside from basic modernity**
<details>

Exploship
<img width="1018" height="676" alt="image" src="https://github.com/user-attachments/assets/5ef13935-00b2-4f78-9363-87ddf1041c78" />

Whiteship_pubby
![image](https://github.com/user-attachments/assets/81052800-5710-4373-9775-9f28a81ccb9d)

Brig (Updated)
<img width="623" height="700" alt="image" src="https://github.com/user-attachments/assets/69d7c434-f5b1-42bc-b21f-502597551b2f" />

Medbay (Updated)
<img width="542" height="415" alt="image" src="https://github.com/user-attachments/assets/31bd83f5-cd95-4096-a3ff-ab0930a0fc41" />

SM Chamber
![image](https://github.com/user-attachments/assets/4f415b7d-04a6-4f07-8e33-50b5cea98e71)

AI Satellite
<img width="645" height="857" alt="image" src="https://github.com/user-attachments/assets/8c6ea8c7-6ddf-4857-ac76-b7ec7fa8e166" />

Cargo 
![image](https://github.com/user-attachments/assets/cb8652d2-bf45-41ba-8956-8ef5420e3f51)

Science
<img width="962" height="735" alt="image" src="https://github.com/user-attachments/assets/d923ed3a-a18b-4f0f-8705-b8ff237ed9dd" />

Bridge
![image](https://github.com/user-attachments/assets/fb6d373d-ffbc-4bd6-9ce5-69136c6976ce)

Monastery
<img width="483" height="818" alt="image" src="https://github.com/user-attachments/assets/7f96e09a-e335-4160-a810-adb59d9b8af5" />

Telecomms/ShipDock
![image](https://github.com/user-attachments/assets/1a7e67bb-79bb-4146-af35-a23f30482587)

Atmospherics
<img width="695" height="604" alt="image" src="https://github.com/user-attachments/assets/04efb3ba-4ec6-46cc-ba12-8615e6283564" />

Syndicate Asteroid (above dorms space)
<img width="939" height="865" alt="image" src="https://github.com/user-attachments/assets/0dff4e57-4021-437a-8b81-f3fbdd387a83" />

Dorms Itself
<img width="837" height="819" alt="image" src="https://github.com/user-attachments/assets/1c3d3039-c384-43d1-bd31-3870a13585ff" />

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

## Changelog
:cl:
add: Added Pubbystation
tweak: Changed Shuttle.dm to Spawn New Pubby Whiteship/redesignated Old WS as Birdboat for Admin Usage
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

Tsar-Salat commented:
Hm

cursedkiller commented:
everything now should be perfectly in order for review or criticism

MarkusLarsson421 commented:
Fix your Changelist.

Something like...
Add: Recreated Pubby.
Add: (Pubby) Created two new ships.

Doesn't need to be super detailed, keep it in the PR body instead.

ClownMoff commented:
![pubbyissues](https://github.com/user-attachments/assets/cce61d5b-7bf8-4363-b847-4bcd1109e9ed)

It's time for reviewerino:
1.- Medbay usually runs 1 crypod, but you could fit those 2 in there. There's no reason to leave that big hole with nothing on it.
2.- This is too cramped, there should not be a 1 tile wide space like that in the center of the treatment area, tons of people need to get through there.
3.- Double wall 🗿
4,- This is too random of a space. Shwoers should not be that far from the treatment area, you want to keep track of your patiences that need to be under a shower. Also that random sleeper and reinforced walls.
5.- This hallway has really almost nothing going for it, too much random filler and gardens that do not look organic at all, why is that capybara garden in that exist there in the middle of the hallway but it's just 3 tiles long? 
6.- Genetics shoud have 2 computers, it should be blue themed since they are no longer part of science, it has a random clone sign even though it's not the cloning room and wh is that weird windoor exit there?
7.-Again, it should be blue themed, it has random reinforced walls, and should have the cloning sign there instead of the radioactive one. Also, the room is too big. There's nothing wrong with giving more space to maintenance.
8.- This room is too small, way too small.- It does not have an APC or air alarm and it should have exits in different points given its location.
9.- Another hallway with barely anything on it
10.- CMO office does not have reinforced walls or shutters.
11.- Double wall 🗿

Unmarked stuff:
Chemistry could use some more sauce/decoration
The psychiatrist room looks like a break room. Give it bookcases or something.
More unique decal work would be appreciated

Overall, still some job to do, but I encourage you to keep working on it. I will also post one for security

ClownMoff commented:
![pubbybrigissues](https://github.com/user-attachments/assets/3d2e263d-20a8-447d-aeff-6cee52161c31)

Brig time:
1.- Shutters and firelocks should go on the same place
2.-  Double wall 🗿
3.- Too bland, give it something more and probably even make it smaller
4.- This makes it too easy to break in and out of the prison, it's a blind spot for security because of the wall there with the SPACE sign and you can just use jaws or open a couple doors and you can get poeple outside of prison pretty easily and without security noticing. Also the prison and processing area for lavand are different punishments, the shuttle should not be there. If the bottom door should be in the brig our outside its another agument since you don't want to depend on officers to leave but you also want to get your stuff back from the brig, If you are going to make it public, those doors need to have a map helper so doors can be opened by anyone in the direction towards the exit.
5.- Too big, too much empty space. Give it a stasis and a rolling bed or something.
6.- I like the idea of this prison. Does it have the reinforced prioson floor? I think the decals are a bit off and dont give the prisoner center vibe but that may just be me. 
7.- These 2 rooms should be together, no reason to have these 2 things in different places.
8.- Is good that this is open to space so people can get stuff back, but putting it that deep into the brig makes it hard to get your stuff back from there.
9.- Too much unused space
10.- Too much unused space

I am not a maintainer but I advice correcting this stuff, I've mapped enough and merged enough mapping stuff to know that someone will eventually end up pointing a lot of this stuff

cursedkiller commented:
fixed up medbay according to specifications also that sleeper by genetics is actually a organ harvester 
![image](https://github.com/user-attachments/assets/876ede86-16ca-4633-9cab-22123a8e527b)

Brig is abit diffrent too now as i tried to spruce up the empty regions , i made the gulag function through visitation instead with a Sec access door and abit more visibility as i specifically wanted it to be near the main part of brig processing instead of a out of sight isolated ship hangar, only thing i didnt really do is 8 and 7 which i feel dont need change as they look nice , function well , and the evidence room is easy to breach as there should be little traffic around the break room/execution
![image](https://github.com/user-attachments/assets/ac1a9a1c-6605-48ea-861a-2f19513cda24)

MarkusLarsson421 commented:
The prison is very weirdly located. For any officer to get to the brig phys room without entering the prison. How long would that take? What would their path look like?

cursedkiller commented:
it takes about 13.27s if your going the full loop(1) or 7.65s if you go through the prison (2) 
in my sec experience security treats genpop as perma usually leaving prisoners in time out for whatever evil crime they commited, resulting in them just sitting by the exit to gloat or shittalk one another

This Brig is more Designed around the Convict and their engagment with the staff (such as the medical area or Gulag for further processing if your prisoner is just too anoyying)
As most sec in other stations use the Transfer room as a shortcut, i intend the same but with the prison but more dangerous (Mechanically this should result in more interactions)

Note: The Shutter Lockdown gates would also force attackers or defenders to go through the prison unless you break them down (3, also the Shutters are modified to be weak)
![image](https://github.com/user-attachments/assets/5759fa18-ef0f-428d-8d4a-bdda8c7cb025)

cursedkiller commented:
i incorporated some small personal quarters and some more stuff for prisoners to do/make the area more homely 
also i realised half the stations just use a fancy shielded area instead of a rad shelter which although kinda iffy i could incorporate since the brig is abit crampt
![image](https://github.com/user-attachments/assets/09f28bb9-3813-48ce-96f4-a24f8d867c80)
Also i invented a coat hanger for the Hos

cursedkiller commented:
Uploaded new Testing Evidence Photos for everything until now

cursedkiller commented on _maps/map_files/PubbyStation/job_changes.dm:
ive replaced the job changes thingy with the map adjustment thing and tested it

cursedkiller replied:
i updated the job changes to the map thing , sorry for the delay

EvilDragonfiend commented on _maps/map_files/PubbyStation/map_adjustment_Pubby:
this is not ".dm" file.
Please check how echo is implemented.

cursedkiller replied:
I just converted the thing into DM i goofed up somehow earlier

EvilDragonfiend reviewed (changes requested):
_maps/map_files/PubbyStation/map_adjustment_pubby.dm
is included in nowhere. chevk how echo is included.
also for the path name, do not use capital (/map_adjustment/pubby_station)

EvilDragonfiend commented on _maps/map_files/PubbyStation/map_adjustment_pubby.dm:
```diff
@@ -0,0 +1,14 @@
+	change_job_access(JOB_NAME_EXPLORATION_CREW, ACCESS_MAINT_TUNNELS)
+	change_job_access(JOB_NAME_HEAD_OF_SECURITY, ACCESS_CREMATORIUM)
+	change_job_access(JOB_NAME_WARDEN, ACCESS_CREMATORIUM)
+	change_job_access(JOB_NAME_SECURITY_OFFICER, ACCESS_CREMATORIUM) // sample code
```
you didn't remove `// sample code` that's what I meant.

cursedkiller replied:
i included everything and i think it should be fine now , i also removed some disposal sortings tags since chap/libr is isolated and other places got space disposals

cursedkiller commented on _maps/map_files/PubbyStation/map_adjustment_pubby.dm:
```diff
@@ -0,0 +1,20 @@
+*/
+
+/datum/map_adjustment/pubby_station
+	map_file_name = "pubbystation.dmm"
```
![image](https://github.com/user-attachments/assets/83fc2c4e-5a5b-4ca0-ae0f-b38e8730448f)
it is capital , or it says it is

EvilDragonfiend replied:
Pubby**_S_**tation.dmm

cursedkiller replied:
oh damn , my bad im dumb

Rukofamicom commented:
Go look at issues, find one that's easy, fix it and ping me to merge it. 
You won't have to keep asking for workflow approvals if you do.

EvilDragonfiend commented on code/datums/map_adjustment_include.dm:
```diff
@@ -8,3 +8,4 @@
 */
 
 #include "..\..\_maps\map_files\EchoStation\map_adjustment_echo.dm"
+#include "..\..\_maps\map_files\EchoStation\map_adjustment_pubby.dm"
```
```suggestion
#include "..\..\_maps\map_files\PubbyStation\map_adjustment_pubby.dm"
```

Rukofamicom commented:
it's okay the other PR merged so it will now autorun checks.

cursedkiller commented:
I thank you for your Linter Test Sacrifice :saluting_face:

spockye reviewed (dismissed):
<details>
<summary>Alien whiteship</summary>
<br>
<img width="736" height="358" alt="image" src="https://github.com/user-attachments/assets/140fd59a-530f-42f3-ad1b-9e377cae8216" />
missing wiring / wiring leading to nothing?

<img width="770" height="583" alt="image" src="https://github.com/user-attachments/assets/57b80c00-8992-4aba-87cb-80e5ef09e60b" />
floating pipe, exhaust doesnt go anywhere. it needs somewhere to send the bad gases!

i dont think it matters that much since we're a roleplay server, but the stuff you can get is a little crazy, with a stetchkin, a frag grenade, full alien surgery tools, laser gun, alien laser gun and a medbeam gun. refer to our balancing man 🫡 

other then that, i like it! good work :)

</details>


<details>
<summary>exploration ship</summary>
<br>
<img width="453" height="695" alt="image" src="https://github.com/user-attachments/assets/5cf32863-26d0-4484-8e49-0a4510f82d77" />
missing pipes, pipes leading to nothing

<img width="456" height="412" alt="image" src="https://github.com/user-attachments/assets/f728ea0d-1d59-4d09-95f7-8d9b85dd2821" />

floating cable

<img width="415" height="403" alt="image" src="https://github.com/user-attachments/assets/54ab301e-df14-4a43-bc19-fa03e761f1a0" />

tiny fan, the works of the devil.

looks alright, not sure what i think about the shutter airlocks at the back, but wont make you change it atm. 

!!!!! why are there two versions of the ship? theres the pubby exploration in the exploration folder, and the pubby exploration in the whiteship folder. there's already a whiteship for pubby. not sure if this was done by mistake? !!!!

</details>

Over to the station itself!

<details>
<summary>piping/wiring/plumbing issues</summary>
<br>
Gonna start by mentioning that theres a whole lot of atmos piping errors. i wont point them all out, but i'll show some examples:
<img width="650" height="666" alt="image" src="https://github.com/user-attachments/assets/2de35362-c505-4339-bc05-50f065569dfd" />

<img width="624" height="589" alt="image" src="https://github.com/user-attachments/assets/3a4f93e1-535a-42b0-ba58-bc3b38eaa9be" />

<img width="774" height="819" alt="image" src="https://github.com/user-attachments/assets/8a205aeb-2067-4807-9c14-811150dc4981" />

<img width="636" height="672" alt="image" src="https://github.com/user-attachments/assets/1fee79c9-343e-46f5-b67c-df57563c2b77" />

<img width="427" height="584" alt="image" src="https://github.com/user-attachments/assets/df2513ae-a25b-4eec-b13a-a1f91e1899ff" />

<img width="538" height="714" alt="image" src="https://github.com/user-attachments/assets/d7f66665-5f8e-4e34-9c15-40048abdb096" />
No disposals in chapel asteroid?
</details>

<details>
<summary>Areas</summary>
<br>
<img width="491" height="396" alt="image" src="https://github.com/user-attachments/assets/ae355c9b-4787-4816-8dd8-7a3ad9f30956" />
extend area

<img width="604" height="625" alt="image" src="https://github.com/user-attachments/assets/072e3a17-d879-4949-8c21-f83fcf974f5b" />
wild area

<img width="604" height="659" alt="image" src="https://github.com/user-attachments/assets/9c8a202e-9e31-496a-a9d1-b06ccbfef17d" />

windows are escaping the area

<img width="571" height="778" alt="image" src="https://github.com/user-attachments/assets/53f694ce-e8e4-40ec-9489-f127d900e905" />

back into containment, area!!

</details>

<img width="623" height="606" alt="image" src="https://github.com/user-attachments/assets/e2907d51-25d2-4719-b0e8-806a1a06016c" />

the syndicate asteroid.
do i like it? hell yes
is it well made? yep!
is it a good idea to put it on the station z?....
i dont know. i feel like this could get heavily abused, but i like the idea! check in with people smarter then me. 

<img width="453" height="415" alt="image" src="https://github.com/user-attachments/assets/d9595588-78f8-484f-9e8b-c8f154e3285f" />
and thats an actual nuke. no. no


------



overall, i like the changes! it definitely brings pubby more alive. theres a little work to be done here and there, but i'd love to see this TM'd, if it hasn't already. good work!

mrmanlikesbt commented:
@cursedkiller i strongly suggest getting head dev approval before putting more work into this

cursedkiller commented:
ill get in contact one of them, but this is more of a passion project regardless if it gets merged or not since this is my first real attempt at mapping

cursedkiller commented:
> I love this and i love you

your lucky github doesnt have a nerd emoji reply function in the avalable options

BarteG44 reviewed (changes requested):
Why use bluespace tiles here if you hide them under decals anyhow
<img width="849" height="645" alt="image" src="https://github.com/user-attachments/assets/9dcd81b7-f982-4293-adb9-5f05114d68cc" />

---
You can use the grass variant with no border if you don't want it spilling out from under the windows
<img width="529" height="769" alt="image" src="https://github.com/user-attachments/assets/65e276c0-e9f2-4df1-8fcb-d70b7c0bd889" />
<img width="559" height="608" alt="image" src="https://github.com/user-attachments/assets/552ead48-8237-4783-8476-56e7c18bd35b" />

---
You gotta climb the table to get across the room
<img width="352" height="256" alt="image" src="https://github.com/user-attachments/assets/856f0b54-fdf3-474b-a7ec-eeea48bd1c69" />

---
Pipe with the wrong "hide" var
<img width="309" height="280" alt="image" src="https://github.com/user-attachments/assets/14ad02ed-1cc9-4f9c-bfaa-e72525c25f56" />

---
This helper has a chance of replacing a door with a wall, making this room completely inaccesible and forcing pipes to go under a wall
<img width="160" height="160" alt="image" src="https://github.com/user-attachments/assets/bab2f0bd-ad69-4036-b94a-18855c3ba798" />

---
Decals on carpets don't look the best
<img width="734" height="381" alt="image" src="https://github.com/user-attachments/assets/6bf76e01-8fa6-489b-9fba-86e3f1a4d45f" />

---
Random filter
<img width="160" height="160" alt="image" src="https://github.com/user-attachments/assets/ac3e9ce0-1ba9-4c52-ae11-a52b75c3f582" />

---
More of the same with pipes
<img width="128" height="160" alt="image" src="https://github.com/user-attachments/assets/6127c545-ef34-47b8-9063-00dc29ea19f1" />
<img width="213" height="246" alt="image" src="https://github.com/user-attachments/assets/f5a6f2c5-773b-444d-835a-5ac450186a5d" />

---
Missing corner
<img width="256" height="160" alt="image" src="https://github.com/user-attachments/assets/789b3442-59aa-4775-bb58-d76f4d0c0b66" />

---
Bit weird to have windows inset into walls like this but sure
<img width="160" height="384" alt="image" src="https://github.com/user-attachments/assets/8669c9f2-f307-4659-9ef9-198105892351" />

---
Doubled up caution here
<img width="96" height="128" alt="image" src="https://github.com/user-attachments/assets/bd29e789-0003-4981-b45a-4e2aa2dd1c74" />

plus one here
<img width="96" height="128" alt="image" src="https://github.com/user-attachments/assets/c1711a31-cb72-4c63-b965-90d3d7950d98" />

---
Misaligned decal
<img width="321" height="354" alt="image" src="https://github.com/user-attachments/assets/2610f413-5d14-46e3-a86a-dd67e163e129" />

---
More cables under walls
<img width="224" height="128" alt="image" src="https://github.com/user-attachments/assets/7487a48b-f24e-41b1-9445-6a639780d7c0" />

---

cursedkiller commented:
> Gave it a quick look before sleep


Cheers for the Feedback it all made me really hate atmospherics
that aside,
ive fixed all the silly errors you pointed out and the issues with the windows so many thanks for that but i did leave some things as is like the:


> grass variant

Personaly the Grass of the Central Monument looks really nice sticking out , but i did correct it in botany and small displays
<img width="501" height="457" alt="image" src="https://github.com/user-attachments/assets/b7e2172d-61bb-4edc-978d-54e45ea03498" />


> Misaligned decal

this singular decal is the bane of my existance and refused to be fixed, i tried


> You tend to make corners out of 2 straight pieces

Honestly when i first started the Project i didnt know how Strong DMM worked or that you could Change "DIR" to make bendy lines so i made them seperate , but now as i tried correcting this oversight i realised that they look way better and i like the jagged nonbendy look more


> Biggest ask is that you should consider updating all the door accesses to use the new access mapping helpers

I had Tried this and Quickly realised i have no idea what these are , i looked at meta , box and card to try copy instances of where they were used and when not completely missing i only really found them in engi, ai sat and bits of service so i did that , Not sure if you meant to do that or try Place these at every single department


> Atmos in general has a bit too many catwalks

Fair, i reduced the amount but i still love using and glazing catwalks
<img width="923" height="818" alt="image" src="https://github.com/user-attachments/assets/96f06a28-7b20-4dd4-a0c6-a60cd1392696" />

BarteG44 commented:
>this singular decal is the bane of my existance and refused to be fixed, i tried

Use guideline_mid_alt. Don't pixelshift decals whenever possible
<img width="746" height="380" alt="image" src="https://github.com/user-attachments/assets/01775d64-9791-40a7-a6e3-ab07f4265462" />

---
>I had Tried this and Quickly realised i have no idea what these are

Just use the helpers under the "any" sybtype. All you have to do is stack all the helpers corresponding to your needed access onto a door, as a replacement to editing the "req_access_one_txt" var. Most doors will require just one helper though.
<img width="420" height="366" alt="image" src="https://github.com/user-attachments/assets/6674e9d8-e234-41f4-8eb3-348101ab3188" />

For example, here's how you'd set up a door that leads to a shared area between science and medbay
<img width="859" height="314" alt="image" src="https://github.com/user-attachments/assets/7b69dd1b-5182-4a5d-8ddf-ce6b6c9115d8" />

it uses the /any/ subtype, so anyone with *either* medical or science access can open it


If you used the helpers from the /all/ subtype, only people with *both* access can open it, so for example a geneticist or captain, but not a doctor or scientist.

Just make sure you clear all of these vars on the door if you use the access helpers on them, and don't mix the /any/ and /all/ subtypes on one door
<img width="416" height="120" alt="image" src="https://github.com/user-attachments/assets/9f15e003-955e-4858-bd34-f22bb3b01599" />

BarteG44 commented:
Foam dispenser under a scrubber
<img width="160" height="160" alt="image" src="https://github.com/user-attachments/assets/e8253531-f6d9-4677-bfd3-fbfeb9dc88c7" />

---
However much i love funky wires, Bacon wants them murdered in favor of smart(dumb)wires. You might need to replace these
<img width="288" height="256" alt="image" src="https://github.com/user-attachments/assets/17d9dab2-8a2c-4016-a4ca-1c78ef9c951e" />

---
3 Abandoned airlock helpers have a very high chance of just blocking off this whole section of maintenance with a wall. Not necessarily a bad thing if you intended this but just beware
<img width="384" height="448" alt="image" src="https://github.com/user-attachments/assets/7c714e23-75d9-4830-ac45-fd01bbeb902d" />

---
Try and not overlap things like lights and cameras when possible
<img width="160" height="128" alt="image" src="https://github.com/user-attachments/assets/3801921a-84ef-4b92-bbea-40dcd2fec599" />
<img width="96" height="96" alt="image" src="https://github.com/user-attachments/assets/5b09edc8-a78e-400e-a726-c3a8d9c701d1" />
<img width="160" height="192" alt="image" src="https://github.com/user-attachments/assets/01dcfec0-d5a6-4e37-8c19-97fb915f87b3" />


---
Window bulb
<img width="160" height="160" alt="image" src="https://github.com/user-attachments/assets/a56a42fe-21ec-45db-8342-badbbb48053e" />

---
Possibly missplaced red arrow decals
<img width="160" height="96" alt="image" src="https://github.com/user-attachments/assets/90810603-3059-46bb-be65-b8acd4111170" />

---
Decal on grass
<img width="224" height="128" alt="image" src="https://github.com/user-attachments/assets/6940868b-dcd9-4973-9fe1-589754a41e46" />

---
Missing decal under a door
<img width="224" height="224" alt="image" src="https://github.com/user-attachments/assets/92dd1e84-75fd-43aa-aac3-544854876029" />

---
These 2 decals have a highier alpha value
<img width="160" height="192" alt="image" src="https://github.com/user-attachments/assets/ed32496f-02a0-483f-8329-58431e728202" />

---
Decal under a window
<img width="160" height="256" alt="image" src="https://github.com/user-attachments/assets/93b781a4-cd40-489a-ae20-f4b31d748cc9" />

BarteG44 commented:
![8398281afb8614e16451609b9d841edd](https://github.com/user-attachments/assets/cffdfbd0-f09b-4859-bf7e-a5521fb0f742)

cursedkiller commented:
> Don't be afraid to reimagine Pubby

Pubby design was sorta outdated in some aspects but i believe i have fixed most of such glaring issues, similiarly i dont really wish to reimagine Pubby in its entirely instead id rather upgrade its features as i wish it to still be Pubby and not just the name alone

Also i Lowkey dislike the modern cartoonish Filler artstyle most modern maps and objects have and thus kinda prefer a the more rustic look

Anywho i revamped dorm to meet modern ERP requirments since i dont got much else to fix i think
<img width="951" height="896" alt="image" src="https://github.com/user-attachments/assets/453fbfd4-cd5e-4d62-83c6-da5e74fb5547" />

cursedkiller commented:
Small revamp of dorms and warden office 
<img width="1018" height="900" alt="image" src="https://github.com/user-attachments/assets/478281e1-295b-491b-b510-fd357b48f970" />

<img width="466" height="315" alt="image" src="https://github.com/user-attachments/assets/f54a046b-2c8b-4da1-9e47-9d1c7680b3cf" />

mrmanlikesbt commented:
Your changelog can be summed up as:

map: Added Pubbystation

All 4 changelog entries you have come with a new map.

Also, don't capitalize the changelog entry type

cursedkiller commented:
> this is the PR causing smooth movement

I spent like 10 hours trying to figure out what the issue was and narrowed it down to this bar counter
<img width="642" height="509" alt="image" src="https://github.com/user-attachments/assets/136a62fa-5eb3-4712-9dba-621861cf473b" />

The Problem has been now fixed after i had removed a bottle, and i got no idea how any of that works or happend in the first place😭

mrmanlikesbt reviewed (changes requested):
Remove or alter the appearance of the Pubby pod

spockye reviewed (changes requested):
<details>
  <summary>i'll start with piping errors: </summary>

<img width="618" height="498" alt="Image" src="https://github.com/user-attachments/assets/ad825ed0-43dc-42a3-b4d8-174e34fe6859" />

missing pipes 

<img width="370" height="408" alt="Image" src="https://github.com/user-attachments/assets/3871ba7b-89fe-4935-95e9-c0b097a13866" />
missing pipes? 

<img width="565" height="454" alt="Image" src="https://github.com/user-attachments/assets/d935d113-39ce-420f-b9c8-a02737408c9f" />

wild pipe 

<img width="608" height="530" alt="Image" src="https://github.com/user-attachments/assets/af8a1c36-8803-43eb-9998-85d62ec7e1f7" />
missing pipe

</details>

<details>
  <summary>cable errors:  </summary>

<img width="636" height="779" alt="Image" src="https://github.com/user-attachments/assets/1998d4b6-cd66-416c-b85f-4c730c0b4ad5" />

<img width="602" height="584" alt="Image" src="https://github.com/user-attachments/assets/5d42ff12-9c9c-4827-866d-920508adec55" />

</details>


----------------------------------------------------------------


So! on to other things.

---decalling---
Your decals are a little all over the place. 

<img width="892" height="908" alt="Image" src="https://github.com/user-attachments/assets/4734b421-b137-48cc-9d08-6c29e0772e81" />

It seems like you've maybe forgotten to use corner decals, and have in other places overlapped decals instead of placing corner decals (see your dark thinplating).
I'm also just in general seeing a lot of inconsistencies everywhere with what gets decalling. I've just pointed out a few I saw in this area, but I'm seeing a lot of the same in all the new sections. I'd recommend you go through and check for yourself and try to fix them. If you want, you can send me a DM and I'll go through the map with you over screenshare.
A lot of maps have a problem with underusing decalling, but I might be leaning towards there being a little too many here for its own good.
---Design---
So firstly, the bridge.
I can't say I'm much of a fan of some of the new changes.
If I were to point out one thing, try to spruce up your flooring a little more. While the grid floors are detailed and nice looking, I do not believe using them as a primary flooring works in their favour. It ends up looking "cluttered." Now, this may as well just be a personal gripe, but I do recommend trying something different.
Cargo
It's nice! It also consists of one type of flooring, though your decalling complements it well and it ends up working out. I would say you could try experimenting with using more turf types, but it's not necessary.
Dorms
Very nice. I like it. No comments as of now :) (other than a missing pool railing)
Medbay
I'm seeing a lot of similar decalling errors as I mentioned earlier. Try going through it, see what you find yourself, and I'll write a longer list of the issues I've noticed if needed. Though in general it is much better than the old version!
Science
Looks good, no notes as of now.
---Final Notes---
So in general, I like the improvements. You've put your own style to it, and I can see the passion! Though this definitely needs a little more time in the oven.
Go through your decalling, check your grids, check your areas. Try to touch on the turfs if you can, and send me a message if you want that detailed breakdown. Good work!

Dorms
very nice. i like it. no comments as of now :)   (other than a missing pool railing)

Medbay
im seeing a lot of similar decalling errors as i mentioned earlier. try going through it, see what you find yourself, and i'll write a longer list of the issues i've noticed if needed. Though in general it is much better than the old version! 

Science 
looks good, no notes as of now 


final notes
so in general, i like the improvements. you've put your own style to it, and i can see the passion! Though this definitely needs a little more time in the oven. 
go through your decalling, check your grids, check your areas. Try to touch on the turfs if you can, and send me a message if you want that detailed breakdown. Good work!

cursedkiller committed:
Reboot  Test

cursedkiller committed:
Trinkets and Baubles

cursedkiller committed:
Last Commit before sleep

cursedkiller committed:
Added library Dock

cursedkiller committed:
Can Now dock in library area

cursedkiller committed:
Add's More lightswitches

cursedkiller committed:
Fixes the Super Duper Party Cannon

cursedkiller committed:
Another quick fix

cursedkiller committed:
Removed Party cannon due to bug

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
Delete _maps/shuttles/escape_pod/PubbyStation.dmm

cursedkiller committed:
Centcom Docking Update

cursedkiller committed:
1000IQ piping fix?

cursedkiller committed:
Pipe Bugfix

cursedkiller committed:
Test fix

cursedkiller committed:
More Atmos fix

cursedkiller committed:
Man im eepy

cursedkiller committed:
loweky dumb pipes

cursedkiller committed:
Mech error what?

cursedkiller committed:
is it fixed now?

cursedkiller committed:
experiment

cursedkiller committed:
1 bug down 5 more to go

cursedkiller committed:
hate these things

cursedkiller committed:
Fully restores Monastery MRMAN atmos

cursedkiller committed:
Last Monastery Dock cycler?

cursedkiller committed:
Dare i say gulag shuttle..?

cursedkiller committed:
i Darn't

cursedkiller committed:
All restored Pre Atmos nerf

cursedkiller committed:
FInal before sleep!!!

cursedkiller committed:
If your power is decals what are you without them

cursedkiller committed:
Linter fix

cursedkiller committed:
One must imagine intergration tests happy

cursedkiller committed:
what is wrong with you..?

cursedkiller committed:
the hell..?

cursedkiller committed:
did that do it ?

cursedkiller committed:
what about now

cursedkiller committed:
Courtroom Revamp

cursedkiller committed:
Smol dcal fix

cursedkiller committed:
Last Atmos Cleanup

cursedkiller committed:
Bug Flavourtext

cursedkiller committed:
Court final fix

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
Spockeyes tile expansion (smol)

cursedkiller committed:
Small office revamp

cursedkiller committed:
Bug fix

cursedkiller committed:
Disposal Fix

cursedkiller committed:
bit more fixes

cursedkiller committed:
Le finish nitpick

cursedkiller committed:
did i commit before..?

cursedkiller committed:
Last second bug brueh

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
Small map linter

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
Pubbuh update

cursedkiller committed:
Reverts Cargo changes abit

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
moves explo bay to monastery

cursedkiller committed:
i Fat fingered all of these linters

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
Who broke areas?

cursedkiller committed:
LETS DO THIS

cursedkiller committed:
Bug fix ?

cursedkiller committed:
Final fix before slep

cursedkiller committed:
Initialising ERP Protocols

cursedkiller committed:
rahh!!!

cursedkiller committed:
let me sleep...

cursedkiller committed:
Assorted Bug fix final

cursedkiller committed:
Large Assorted Bug Fix

cursedkiller committed:
Last Nicpicks

cursedkiller committed:
Silly active turfs

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
Small fix

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
I Metagrudge whoever broke access's

cursedkiller committed:
Another Patch

cursedkiller committed:
Final Patch (I hate Access Helpers)

cursedkiller committed:
Fix..?

cursedkiller committed:
Smart Wire Overhaul

cursedkiller committed:
Smart Wire Overhaul

cursedkiller committed:
Merge branch 'master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
Smart Wire fix's

cursedkiller committed:
Last Small Patches

spockye commented:
will review soon (tm)

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
cant playtest

cursedkiller committed:
sign fix

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
something wierd

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
Who keeps adding bloat

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
Stop breaking my stuff brueh

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
Day 253 of no TM smh

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
Literaly Just updated it just for Another Update..

cursedkiller committed:
New cigars lowkey ugly

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
breuh

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
Bee station or B Station ?

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
This is getting grating

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
mapdm fix

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
Actually a Error this time

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
Fixes Brig Cells (+SYNC)

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
Recompiles one final time (just fixes the error)

cursedkiller committed:
Fixes a Leaky SM Import Pipe

cursedkiller committed:
Fixed ALOT of bugs

cursedkiller committed:
something broke

cursedkiller committed:
was this it ?

cursedkiller committed:
this ?

cursedkiller committed:
There now?

cursedkiller committed:
Fixes Dumbbpit

cursedkiller committed:
fix ?

cursedkiller committed:
final fix

cursedkiller committed:
smol fix 2

cursedkiller committed:
i forgot to save....

cursedkiller committed:
Small Additional QOL

cursedkiller committed:
Fixes a Few Minor power issues/cable bugs

cursedkiller committed:
Sneaky ghost APC linter , what?

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
Where are they popping up from this time?

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
Giant Anoyyance Update

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
Science Slop PR got merged....

cursedkiller committed:
wierd error

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
Doublechecking if limbgrower is my fault

cursedkiller committed:
Somebody Brickked Roundstart Limbgrower

cursedkiller committed:
more Fluff and Fixes a few wiring complications

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
File blocks dislike eachother

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
Still works?

cursedkiller committed:
Small Qol Features

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
updates Rags

cursedkiller committed:
Adds Alot more Wall Fluff

cursedkiller committed:
1 linter

cursedkiller committed:
All the bugs i found during TM

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
Nerfs party button slightly (Per Request)

cursedkiller committed:
Global Pipenet Fix (NT-SMU Edition)

cursedkiller committed:
Further Atmos Improvments

cursedkiller committed:
Additional Tweak

cursedkiller committed:
Bugs from last round

cursedkiller committed:
Add's ERP (Raspberry Forced me)

cursedkiller committed:
Small pipe ifx

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
Series of Small fixes

cursedkiller committed:
Linter fix

cursedkiller committed:
Simplified Monastery SMES due to Anomoly

cursedkiller committed:
Re-Designed Monastery Powergrid abit

cursedkiller committed:
Adds Wheelchair Accessability

cursedkiller committed:
Tiny fixes i noticed

cursedkiller committed:
Xenoarch fixes (NTS-Shuttle)

cursedkiller committed:
Xenoarch full fix

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
Linter fix

cursedkiller committed:
Readds Pubby Whiteship

cursedkiller committed:
More improvments!

cursedkiller committed:
shuttle fix 2

cursedkiller committed:
FINALLY

cursedkiller committed:
Removes abit of light fixtures and Other QOL

cursedkiller committed:
Nerfs Lights around the station

cursedkiller committed:
Sink fix

cursedkiller commented:
I cut some of the More redundant Lights and chipped off a few KW of energy consumption but some lights are just too pretty to delete so reducing the electric bill for lights will be a WIP

cursedkiller committed:
forgot to save by accident

cursedkiller committed:
RASPBERRY CAMERA TESTING

cursedkiller committed:
Reverts Raspberrys test

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
Fixes Syndi's winning all the time

cursedkiller committed:
so tired

cursedkiller committed:
im too sleepy...

cursedkiller committed:
Screw this im going to sleep

cursedkiller committed:
More Base TP Proofing

cursedkiller committed:
should be it ?

cursedkiller committed:
is tripping ?

cursedkiller committed:
Just HOW!!!

cursedkiller committed:
now?

cursedkiller committed:
what about now?

cursedkiller committed:
what about this?

cursedkiller committed:
Last crashout before nuke

cursedkiller committed:
NUKED!!!

cursedkiller committed:
TOTAL ANNIHILATION!!!!!!!!

cursedkiller committed:
Oh shit HUGE BUG

cursedkiller committed:
One final tidbit

cursedkiller committed:
what about this?

cursedkiller committed:
close shutters..?

cursedkiller committed:
Temporary Fix for TP

cursedkiller committed:
Bluespace Anchor testing

cursedkiller committed:
IS IT REAL?

cursedkiller committed:
i really hate this

cursedkiller committed:
POSSIBLE?

cursedkiller committed:
ITS ALIVE!!!

Isaacnml committed:
i hate  horologium

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
Thanks to Rasp + Smellody

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
Extra Decal/fluff

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
Added cell charger to jail

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
Added more Fluff and reduced lights

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
Holopad + STOP THE MAPDIFS

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
Additional fixes

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
I hate mapdifs honestly

cursedkiller committed:
small oversight

cursedkiller committed:
this should work?

cursedkiller committed:
there

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
Merge branch 'master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
Update flags.dm

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
Mapdiff bypass

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
small fix

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
Pubbuh extra fluff

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff

cursedkiller committed:
Added fluff + mapdif

cursedkiller committed:
Merge branch 'BeeStation:master' into Top-Secret-Confidential-Stuff
